### DISCUSSION TITLE
[BUG] 无限显示 Network Error 且无法创建邮件地址

### DISCUSSION THREAD

**Discussion by ghost** [dreamhunter2333/cloudflare_temp_email#402] [General]
## 复现步骤

按照文档步骤部署后进入 Admin 管理面板，点击任意选项卡都会出现 Network Error 的报错。

<img width="1476" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/146422759/5d314a24-0e7c-4710-94ed-c3938b5ec4d8">

同时无法创建邮件地址，后缀显示无数据。

<img width="751" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/146422759/1790ef5f-f13f-4574-93c5-fe6ca22077ea">

（当然也可能是我用错了）

打开开发者模式，出现了极其密集的报错，大多数 `ERR_FAILED` 和 CORS 策略问题。

<img width="1476" alt="image" src="https://github.com/dreamhunter2333/cloudflare_temp_email/assets/146422759/2dd8ff20-da2e-43a8-8005-7f07283de930">

## 预期行为

可以创建新的邮箱，且不出现报错。

## 部署方式

- [ ] cli 部署
- [x]  用户界面部署（this one）
## 浏览器环境

macOS Sonoma ，Arc 浏览器

**Comment by dreamhunter2333** [comment]
你好，看了下后端 worker  没有部署成功

打开https://mail.okusa.top/ 显示了错误

**Comment by ghost** [comment]
> 你好，看了下后端 worker 没有部署成功
> 
> 打开https://mail.okusa.top/ 显示了错误

emmm，重新部署了几次，但是问题依然存在。

翻了一下日志，显示是 Incorrect type for Promise: the Promise did not resolve to 'Response'.

求大佬解惑 qaq

**Comment by dreamhunter2333** [comment]
> > 你好，看了下后端 worker 没有部署成功
> 
> > 
> 
> > 打开https://mail.okusa.top/ 显示了错误
> 
> 
> 
> emmm，重新部署了几次，但是问题依然存在。
> 
> 
> 
> 翻了一下日志，显示是 Incorrect type for Promise: the Promise did not resolve to 'Response'.
> 
> 
> 
> 求大佬解惑 qaq

是用户界面上传文件来部署的么，复制粘贴可能会有问题

**Comment by catxian** [comment]
NetworkError这个错误只出现在前端pages设置中自定义域名打开才会出现，默认生成的打开就没有这个错误，能给解决方法吗！

**Comment by dreamhunter2333** [comment]
NetworkError 请检查网络环境和浏览器是否有 dns 或者证书缓存
